Makefile

```diff
@@ -131,12 +131,14 @@ perlcritic: tools/lib/
 .PHONY: test-unused-modules-changed
 test-unused-modules-changed:
 	@echo "[make] Unused modules check called over modified/new files only. For a full run use make test-unused-modules-full"
-	tools/detect_unused_modules -m `git --no-pager diff --name-only --diff-filter=d origin/master | grep '^tests/*' | grep -v '^tests/test_pods/'`
-	tools/detect_unused_modules -m `git --no-pager diff --unified=0 origin/master products/* | grep -oP "^-.*loadtest.*[\"']\K[^\"'].+(?=[\"'])"`
-	tools/detect_unused_modules -m `git --no-pager diff --unified=0 origin/master schedule/* | grep -oP "^-\s+- [\"']?\K.*(?=[\"']?)" | grep -v '{{'`
-
-.PHONY: test-unused-modules
-test-unused-modules:
+	tools/detect_unused_modules -m ` (\
+	git --no-pager diff --name-only --diff-filter=d origin/master | grep '^tests/*' | grep -v '^tests/test_pods/'; \
+	git --no-pager diff --unified=0 origin/master products/* | sed -n "s~^-.*loadtest\s\+\([\"']\)\([^\"']\+\)\1.*~tests/\2.pm~p"; \
+	git --no-pager diff --unified=0 origin/master schedule/* | sed -n "s~^-\s\+-\s\+\([\"']\)\([^\"']\+\)\1.*~tests/\2.pm~p" | grep -v '{{' ) \
```

Comment:
I still wonder why it's necessary to make the regex so complex now

Reply:
Not sure how to make it simpler. IMHO sed does not allow to use `\1` in `[^]`. Also if we join these two together it'd be even more complex.

Reply:
Well, why change the regex from its former place at all? What problems did you encounter?

Reply:
ok, cool